build/MUX-Release.yml

```diff
@@ -2,25 +2,49 @@ name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
 variables:
   minimumExpectedTestsExecutedCount: 35  # Sanity check for minimum expected tests to be reported
   Packaging.EnableSBOMSigning: true
+
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: refs/heads/main
+  - repository: templates_onebranch
+    type: git
+    name: OneBranch.Pipelines/GovernedTemplates
+    ref: refs/heads/main
 jobs:
+- job: Scanning
+  pool: 
+    name: WinDevPool-S
+    demands: ImageOverride -equals WinDevVS16-9
+
+  steps:
+
+  - checkout: self
+
+  - task: NodeTool@0
+
+  - template: v2/Steps/PackageES/Windows.SDL.Sources.Analysis.OS.Undocked.yml@templates_onebranch
+    parameters:
+      globalsdl:
+        codeql:                                          # run codeql
```

Comment:
The comment says "run codeql" but you've got "enable: false".  Is that intended?

Reply:
Looks like it supports Javascript, TypeScript, Python and Powershell. We only use powershell, so I've enabled that.